test-lib/test.js

```diff
@@ -1,15 +1,25 @@
 const fs = require('fs-extra');
 const path = require('path');
+const { stripIndent } = require('common-tags');
 
-if (!fs.existsSync(path.join(__dirname, '/../test/node_modules'))) {
-  fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
-  fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
-}
+fs.removeSync(path.join(__dirname, '/../test/node_modules'));
```

Comment:
I needed this simulated node_modules folder to be less fake than before, and that required a more robust tear-down and setup procedure.